corehq/apps/registration/forms.py

```diff
@@ -9,12 +9,34 @@
 from corehq.apps.domain.models import Domain
 from corehq.apps.domain.utils import new_org_re, website_re
 from corehq.apps.orgs.models import Organization
-from corehq.apps.style.forms.widgets import Select2Widget
+from corehq.apps.style.forms.widgets import BootstrapDisabledInput, Select2Widget
 from django.utils.encoding import smart_str
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext_lazy as _
 
-class NewWebUserRegistrationForm(forms.Form):
+class DomainRegistrationForm(forms.Form):
```

Comment:
just moving this; the only change here is dropping the colon after "Project Name"